functions/backup.sh

```diff
@@ -133,14 +133,13 @@ amanda_setup() {
   fi
 
   /usr/sbin/exim --version 2>/dev/null >/dev/null
+  local adminmail="root@$(/bin/hostname)"
```

Comment:
While `local` declarations and assignments on the same line should mask error codes. This line should be above the previous line to make absolutely clear that we are assessing the return code for `/usr/sbin/exim` and not `/bin/hostname` in the next few lines.